feature-libs/organization/account-summary/occ/model/index.ts

```diff
@@ -0,0 +1 @@
+export * from './occ-account-summary-endpoints.model';
```

Comment:
this `index.ts` can be removed. (It is better removed it now. When upgrading to Angular 14, if `index.ts` and `public_api.ts` both exist, we use `public_api.ts`, and remove `index.ts`.